website/routes.py

```diff
@@ -1227,6 +1228,13 @@ def make_url_map(app):
         ),
 
         Rule(
+            '/settings/subscribe/',
```

Comment:
Does this mean I have to separate the logic for creating and updating a subscription into two separate functions? Currently, it's all handled by the `subscribe` function: 
https://github.com/samanehsan/osf.io/blob/feature/notifications/website/notifications/views.py#L10